src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -124,7 +124,8 @@ public static class ScriptConstants
         public const string FeatureFlagEnableDebugTracing = "EnableDebugTracing";
         public const string FeatureFlagEnableMultiLanguageWorker = "EnableMultiLanguageWorker";
         public const string FeatureFlagEnableProxies = "EnableProxies";
-        public const string FeatureFlagEnableLinuxEPExecutionCount = "EnableLinuxFEC";
+        public const string HostingConfigEnableLinuxEPExecutionCount = "EnableLinuxFEC";
+        public const string HostingConfigEnableLinuxEPFastLog = "EnableLinuxEPFastLog";
```

Comment:
nit: I'd remove the EP bit - it's not really specific to EP. Also, might rename to EnableLinuxLogBackoff?

Reply:
Removed EP portions. While the Platform functionality is EP specific (scaling), the host func is not.